src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -862,6 +862,18 @@ public List<Measure> getQiCoreMeasuresByCmsId(Integer qdmCmsId) {
   }
 
   public void validateCmsIdAssociation(String username, Measure qiCoreMeasure, Measure qdmMeasure) {
+    if (!qiCoreMeasure.getModel().equals(ModelType.QI_CORE.getValue())
+        && !qiCoreMeasure.getModel().equals(ModelType.QI_CORE_6_0_0.getValue())) {
+      log.info("CMS ID could not be associated. Must pass in one QDM and one QI-Core measure");
+      throw new InvalidRequestException(
+          "CMS ID could not be associated. Must select one QDM and" + " one QI-Core measure.");
```

Comment:
Not sure if the formatter did something weird, but the concatenation operation isn't required.